### PR TITLE
[FEATURE] Add sorting modal to program package plan

### PR DESCRIPTION
**Pull request by jhihruei**
- Add sorting modal to `/admin/program-packages/<program_id>?tab=sales`.
- Fix plans list didn't order by position.
- Remove original position admin layout for editing plan position.

### CODE REVIEW

File: src/pages/ProgramPackageAdminPage.tsx [hunk 1]
```diff
@@ -134,49 +138,80 @@ const ProgramPackageAdminPage: React.FC = () => {
...
+                        {formatMessage(commonMessages.ui.subscriptionPlan)}
+                      </Button>
+                    </div>
+                  )}
+                />
+                <ItemsSortingModal
+                  items={
+                    // TODO: Remove frontend sorting if it could be handled in backend
+                    programPackage?.plans
+                      .sort((a, b) => a.position - b.position)
```
Comment (jhihruei) [inline src/pages/ProgramPackageAdminPage.tsx:188]: I think handle this sorting in backend is better, but I couldn't edit backend.

Comment (sunnylochun) [reply src/pages/ProgramPackageAdminPage.tsx:188]: why need sort ?
programPackage.plans 
already order by position ascending
look: src/hooks/programPackage.ts: L322

Comment (jhihruei) [reply src/pages/ProgramPackageAdminPage.tsx:188]: Take [this program-packages](https://demo.lodestar-dev.cc/admin/program-packages/d7a3bcdf-fd87-466c-897f-da0e3af4644a?tab=sales) as example 
The position of `programPackage?.plans` is not order by position.
截圖 2022-08-24 18 30 45

Comment (jhihruei) [reply src/pages/ProgramPackageAdminPage.tsx:188]: But thanks for the reminder, I'll check if we could fix this issue by editing gql

**Review by clothe09986 — APPROVED** [review]

File: src/pages/ProgramPackageAdminPage.tsx (hunk 1 above)
Comment (sunnylochun) [reply src/pages/ProgramPackageAdminPage.tsx]: you are right, it's not order by position
but i find the issue of this
I know why the graphql order by position not work

```
// original gql
// useProgramPackage
query GET_PROGRAM_PACKAGE($id: uuid!) {
  program_package_by_pk(id: $id) {
    ...
    program_package_plans(order_by: {position: asc}, {created_at: asc} ) {
       ...
      position
    }
  }
}
```

you can edit order by condition to bellow and try again
```
program_package_plans(order_by: [ {position: asc}, {created_at: asc} ]} {
```

BTW we use [hasura](https://hasura.io/) of our database server

**Review by sunnylochun — APPROVED** [review]

**Pull request merged**